### ISSUE TITLE
Delete the current name content with backspace

### ISSUE THREAD

**Issue by vetrimano** [podio/jquery-mentions-input#139]
I love the library, It would be great if we could have an enhancement where once the name is tagged, if we hit backspace along with the mentions being deleted, i'd like the name (text box content) to also be deleted. Similar to the facebook mentions...

**Comment by wunsjoshi** [comment]
Yes. I am agree with @vetrimano that single backspace key should remove text added using @mentioned.
